fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionFragment.java

```diff
@@ -49,11 +58,19 @@ public class ElectionFragment extends Fragment {
   private final EnumMap<EventState, Integer> actionTextMap = buildActionTextMap();
   private final EnumMap<EventState, Boolean> actionEnablingMap = buildActionEnablingMap();
 
-  public static ElectionFragment newInstance() {
-    return new ElectionFragment();
-  }
+  private final CompositeDisposable disposables = new CompositeDisposable();
```

Comment:
I don't see how you deal with that disposable when the fragment is paused/resumed/destroyed. Am I missing something ?

Reply:
I might have forgotten to dispose of it